playbooks/common/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,30 @@
+---
+# NOTE: ideally this would be just part of a single play that lists the existing
+# checks when openshift_checks is not set or run the requested checks. However,
+# to actually run the checks we need to have the included dependencies to run
+# first and that takes time. To speed up listing checks, we use a separate play
+# that runs before the include to save time and improve the UX.
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
@sosiouxme, in trying to change from OSEv3 to oo_all_hosts (empty group before initialization), I realized we could as well only run this play in localhost. The downside is it will not play nice with host-specific variables.
WDYT?